test/bats/test.bats

```diff
@@ -29,13 +29,13 @@ SLEEP_TIME=1
     run kubectl apply -f ./library/default/samples/constraint.yaml
     assert_success
     sleep 5
-    run kubectl run cosign-demo --namespace default --image=registry:5000/cosign:signed
+    run kubectl run cosign-demo-key --namespace default --image=registry:5000/cosign:signed
```

Comment:
@susanshi i originally added these tests. however, due to #135 we can only 1 cosign verifier defined. 

I can update the config during the tests to remove the key specific verification, if that's preferred?

Reply:
ah i see, yes, Ratify currently don't have a clean way to work with multiple cosign verifiers. Updating the verifier config sounds good for now, you ll find samples for enabling/disalbing the cosign verifier in existing test [here ](https://github.com/deislabs/ratify/blob/main/test/bats/test.bats#L32)

Reply:
@susanshi  added tests. tests in this PR will not pass due to `id-token` [restrictions for forks](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) since it'll fail to sign using GitHub OIDC token.

it passes here which is the same branch on my fork: https://github.com/sozercan/ratify/pull/3

Reply:
@sozercan as your link displayed, the maximum access for PR from public forked repo is `read.` Even we merge the PR, those jobs might fail for following PRs from forked repo, is that correct?

Reply:
@binbin-li that's a good point, this is similar to the aks test that uses `id-token`. looks like it is skipped by default, how is that handled?

Reply:
wdyt about moving this as part of aks tests since we use `id-token` there anyway?

Reply:
aks test is run upon push/workflow_dispatch events for now. I'm ok to move it to aks tests, but the aks tests don't build the images but pull the images from wabbitnetworks ACR for the test, which diff a bit from local registry test suite.

Reply:
can we push an already keyless signed image to wabbitnetworks then so we can use that to validate in non-aks tests? will you be able to push to wabbitnetworks?

`COSIGN_EXPERIMENTAL=1 cosign sign wabbitnetworks.azurecr.io/test/cosign-image:signed-keyless`

Reply:
Hi Sertac, thanks for adding cli test for cosign keyless. Are you planning to to add test to both the [Azure test](https://github.com/deislabs/ratify/blob/main/test/bats/azure-test.bats) and test.bats[local test](https://github.com/deislabs/ratify/blob/main/test/bats/azure-test.bats)? i have shared a token with you through direct message so you have access to test/cosign-image